src/plugins/api/qmlpluginapi.cpp

```diff
@@ -374,5 +374,16 @@ void PluginAPI::registerQmlTypes()
 
     qmlTypesRegistered = true;
 }
+
+MuseScoreCore* PluginAPI::msc() const
+{
+    static MuseScoreCore mscStatic;
+    if (this->context() && this->context()->currentNotation()) {
+        mscStatic.setCurrentScore(this->context()->currentNotation()->elements()->msScore());
```

Comment:
This is still not ideal, as we're storing the pointer to the current Score potentially beyond its lifetime. Probably should change to shared_ptr, or better still get rid of MuseScoreCore completely if possible and just use Score?  Did try making MuseScoreCore injectable but that didn't seem to work.
But until the plugin mechanism is implemented in MU4 it doesn't really matter, none of this is used for anything.